news/2017-10-08-community-mentorship-program-winter-2017-signups-now-open.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
move the banner from "banners" dir to an article-specific folder in wiki/shared using the article filename as the folder, banners dir is for generic or multiuse banners only

Reply:
And that's why i told Naxess to make a generic banner smh

moved it properly this time i think - also fixed the rest